modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx

```diff
@@ -21,6 +22,23 @@ import openDefaultFailureToast from '../../utils/openDefaultFailureToast';
 import openDefaultSuccessToast from '../../utils/openDefaultSuccessToast';
 import {IFDSAction} from '../Actions';
 
+const ACTION_METHOD = {
+	DELETE: 'DELETE',
+	GET: 'GET',
+	PATCH: 'PATCH',
+	POST: 'POST',
+};
+
```

Comment:
Changed due to https://github.com/liferay-frontend/liferay-portal/pull/3765#discussion_r1369886284